simulate/simulate.go

```diff
@@ -501,3 +514,33 @@ func RemoveAncestors(filename string, tree *expandedTree.ETree, outputFilename s
 	outFile = outputFilename
 	fasta.Write(outFile, newFastas)
 }
+
```

Comment:
in seqContainsDels and codonContainsDels, you pass dna.Gaps as an argument. I'm not sure if there is a "correct" thing is to do here, but an alternative is to just get rid of the del argument on lines 521 and 531 and instead write " if base == dna.Gaps {" which will do the same thing without the argument being there.

Reply:
Yes, I second this suggestion.  There might be a typo in the comment above where I think it should be dna.Gap (not plural)

Reply:
Great suggestion, that looks much cleaner. Thanks for the feedback!